universal-sentence-encoder/src/index.ts

```diff
@@ -54,13 +59,15 @@ export class UniversalSentenceEncoder {
   private model: tf.GraphModel;
   private tokenizer: Tokenizer;
 
-  async loadModel() {
-    return tf.loadGraphModel(`${BASE_PATH}model.json`);
+  async loadModel(basePath: string = BASE_PATH) {
+    return tf.loadGraphModel(`${basePath}model.json`);
   }
 
-  async load() {
-    const [model, vocabulary] =
-        await Promise.all([this.loadModel(), loadVocabulary()]);
+  async load(basePath: string = BASE_PATH) {
```

Comment:
instead of basePath, allow user specify the full path, call it modelUrl and vocabUrl. This will be align with other models.

Reply:
@pyu10055 Perfect! Do you have any specific links? Will dig around the API to look for some conventions to follow and update PR hopefully tomorrorw

Reply:
Here is an example of ModelConfig interface of posenet load API.
https://github.com/tensorflow/tfjs-models/blob/master/posenet/src/posenet_model.ts#L111